packages/mui-base/src/useMenu/useMenu.ts

```diff
@@ -37,20 +37,28 @@ const FALLBACK_MENU_CONTEXT: DropdownContextValue = {
  * - [useMenu API](https://mui.com/base-ui/react-menu/hooks-api/#use-menu)
  */
 export function useMenu(parameters: UseMenuParameters = {}): UseMenuReturnValue {
-  const { listboxRef: listboxRefProp, onItemsChange, id: idParam } = parameters;
+  const {
+    listboxRef: listboxRefProp,
+    onItemsChange,
+    id: idParam,
+    disabledItemsFocusable = true,
```

Comment:
This option was needed so that we can keep a backward compatible API on Material UI, by default it works as before.

Reply:
Same applies for the `disableListWrap` prop below.